src/pages/form-validation/list/index.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useCallback, useState } from 'react';
+import {
+    AppBar,
+    Card,
+    Hidden,
+    IconButton,
+    InputProps,
+    Switch,
+    TextField,
+    Toolbar,
+    Typography,
+} from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import { InfoListItem } from '@pxblue/react-components';
+import List from '@material-ui/core/List';
+import ReportIcon from '@material-ui/icons/Report';
+import { Dns } from '@material-ui/icons';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    containerWrapper: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'start',
+        flex: '1 1 0',
+    },
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        backgroundColor: 'white',
+        maxWidth: 800,
+        marginRight: theme.spacing(4),
+        marginLeft: theme.spacing(4),
+        marginTop: theme.spacing(4),
+        width: '100%',
+        [theme.breakpoints.down('xs')]: {
+            margin: 0,
+            minHeight: 'calc(100vh - 56px)',
+        },
+    },
+    passwordCriteria: {
+        paddingTop: 0,
+        paddingBottom: 0,
+    },
+    appbarRoot: {
+        padding: 0,
+    },
+    toolbarGutters: {
+        padding: '0 16px',
+    },
+    skinnyInput: {
+        paddingTop: 11,
+    },
+}));
+
+export const ListFormValidation = (): JSX.Element => {
+    type OnChangeHandler = InputProps['onChange'];
+
+    const theme = useTheme();
+    const classes = useStyles(theme);
+    const [ip, setIp] = useState('10.0.0.1');
+    const dispatch = useDispatch();
+
+    const onNewPasswordChange: OnChangeHandler = useCallback(
+        (event) => {
+            setIp(event.target.value);
+        },
+        [setIp]
```

Comment:
```suggestion
        []
```

The setXXX functions from a useState hook are guaranteed by React to never change, so they can be omitted from the dependencies array.